### ISSUE TITLE
Quadratic Regression

### ISSUE THREAD

**Issue by fditraglia** [leifeld/texreg#80]
It appears that texreg does not appropriately escape expressions contained within `I()` for LaTeX output. For example, consider the following quadratic regression:
```
myreg <- lm(mpg ~ disp + I(disp^2), mtcars)
library(texreg)
texreg(myreg)
```
If the preceding code chunk is placed inside an `.Rmd` file with chunk options set to `results = 'asis'`, it will not compile to LaTeX correctly, since the `^` is not appropriately escaped. This problem does not arise with html output. Here is the error log for `TeX`:

> processing file: texreg-problem.Rmd
>  |................................                                 |  50%
>  ordinary text without R code
>
>  |.................................................................| 100%
> label: unnamed-chunk-1 (with options) 
> List of 2
> $ results: chr "asis"
> $ message: logi FALSE
>
> Version:  1.36.23
> Date:     2017-03-03
> Author:   Philip Leifeld (University of Glasgow)
>
> Please cite the JSS article in your publications -- see citation("texreg").
>
> output file: texreg-problem.knit.md
>
> /usr/lib/rstudio/bin/pandoc/pandoc +RTS -K512m -RTS texreg-problem.utf8.md --to latex --from markdown+autolink_bare_uris+ascii_identifiers+tex_math_single_backslash --output texreg-problem.tex --template /usr/local/lib/R/site-library/rmarkdown/rmd/latex/default-1.17.0.2.tex --highlight-style tango --latex-engine pdflatex --variable graphics=yes --variable 'geometry:margin=1in' --variable 'compact-title:yes' 
> ! Missing $ inserted.
> <inserted text> 
>                $
> l.144 I(disp^
>             2)   & $0.00^{**}$   \\ 
>Here is how much of TeX's memory you used:
> 12652 strings out of 493029
> 182490 string characters out of 6136234
> 283640 words of memory out of 5000000
> 16021 multiletter control sequences out of 15000+600000
> 20617 words of font info for 38 fonts, out of 8000000 for 9000
> 1141 hyphenation exceptions out of 8191
> 31i,10n,35p,263b,282s stack positions out of 5000i,500n,10000p,200000b,80000s
>
> Error: Failed to compile texreg-problem.tex. See texreg-problem.log for more info.
> Execution halted

**Comment by kanxuUCHI** [comment]
The `replaceSymbols` function [https://github.com/leifeld/texreg/blob/5efd64a7224071b2b06154b012fba1b696c48918/R/internal.R#L333] seems to the one they used to latex the row names but surprisingly it's not found anywhere else.

**Comment by fditraglia** [comment]
Interesting. So maybe this is a pretty easy fix...

**Comment by leifeld** [comment]
Fixed in commit 5a21d832132b2be6cca33035044f6244f668ecb8.
